presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -223,6 +224,15 @@ default ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTabl
         return emptyMap();
     }
 
+    /**
+     * Gets the metadata for all columns that match the specified table prefix. Returns a {@link ListTableColumnsResult}
```

Comment:
The method does not return ListTableColumnsResult. It returns stream of them.

same for other methods